src/ctim/schemas/bundle.cljc

```diff
@@ -11,60 +11,67 @@
             [ctim.schemas.sighting :refer [StoredSighting]]
             [ctim.schemas.ttp :refer [StoredTTP]]
             [ctim.schemas.verdict :refer [StoredVerdict]]
-            [schema-tools.core :as st]
-            [schema.core :as s]))
+            #?(:clj  [flanders.core :as f :refer [def-entity-type def-map-type]]
+               :cljs [flanders.core :as f :refer-macros [def-entity-type def-map-type]])))
 
-(s/defschema TypeIdentifier
-  (s/enum "bundle"))
+(def TypeIdentifier
+  (f/eq "bundle"))
 
-(s/defschema ReferenceList [c/ID])
+(def ReferenceList (f/seq-of c/ID
+                             :description "List of references"))
 
-(s/defschema Objects
-  (st/optional-keys
-   {:actors [StoredActor]
-    :campaigns [StoredCampaign]
-    :coas [StoredCOA]
-    :exploit-targets [StoredExploitTarget]
-    :feedbacks [StoredFeedback]
-    :incidents [StoredIncident]
-    :indicators [StoredIndicator]
-    :judgements [StoredJudgement]
-    :sightings [StoredSighting]
-    :ttps [StoredTTP]
-    :verdicts [StoredVerdict]}))
+(def objects-entries
+  (f/optional-entries
+   (f/entry :actors [StoredActor])
+   (f/entry :campaigns [StoredCampaign])
+   (f/entry :coas [StoredCOA])
+   (f/entry :exploit-targets [StoredExploitTarget])
+   (f/entry :feedbacks [StoredFeedback])
```

Comment:
(f/etnry ...) could be replaced with [...], like

```clojure
[:coas [StoredCoa]
  :comment "this is a stored COA"]
```

or we could just alias f/entry with 'e as in

```clojure
(e :coas [StoredCoA]
 :comment "This is a stored COA")
```

It all sucks, just in different ways.